node/pkg/driver/node.go

```diff
@@ -475,14 +475,24 @@ func (d *NodeService) NodeGetInfo(ctx context.Context, req *csi.NodeGetInfoReque
 	logger.Debugf(">>>> NodeGetInfo: called with args %+v", *req)
 	defer logger.Debugf("<<<< NodeGetInfo")
 
-	iscsiIQN, err := d.NodeUtils.ParseIscsiInitiators("/etc/iscsi/initiatorname.iscsi")
-	if err != nil {
-		return nil, status.Error(codes.Internal, err.Error())
+	iscsiIQN, iscsi_err := d.NodeUtils.ParseIscsiInitiators("/etc/iscsi/initiatorname.iscsi")
+	fcList, fc_err := d.NodeUtils.ParseFCPortsName("/sys/class/fc_host/host*/port_name")
+
+	if iscsi_err != nil && fc_err != nil {
```

Comment:
its none standard to run 2 commands and check there error after.
better to run command and respond to its error line after. Better for clarify and reduce bugy areas.

if no iqn it should not fail OR you can return found as bool or just return empty string.  I think that error should indicate error.